client/web/src/site-admin/SiteAdminCreateUserPage.tsx

```diff
@@ -147,7 +165,7 @@ export class SiteAdminCreateUserPage extends React.Component<RouteComponentProps
                                 aria-describedby="site-admin-create-user-page__form-email-help"
                             />
                             <small id="site-admin-create-user-page__form-email-help" className="form-text text-muted">
-                                Optional verified email for the user.
+                                Optional email for the user - verification will be required if provided.
```

Comment:
I find it rare to use `-` in in-product descriptions
```suggestion
                                Optional email for the user that will require verification when provided.
```